src/erldns_event_handler.erl

```diff
@@ -35,7 +37,10 @@ handle_event({_M, start_servers}, State) ->
   case State#state.servers_running of
     false ->
       % Start up the UDP and TCP servers
-      lager:info("Starting the UDP and TCP supervisor"),
+      ?LOG_INFO(#{
+	 log => event, in => start_servers, 
+	 text => "Starting the UDP and TCP supervisor"
+	}),
```

Comment:
```suggestion
      ?LOG_INFO(#{
	 module => erldns_event_handler, function => start_servers, 
	 text => "Starting the UDP and TCP supervisor"
	}),
```

What do you think of switching to common Erlang terms rather than the more abstract `log` and `in`?

Reply:
as far we are consistent across the board, don't see any problems using the module/function keywords.

Reply:
the only caveat is that "log" in the context above is used to differentiate the log type which is normally command or event, so we can keep the log and add module or replace "at" that we use for the module:fun/arity content:

```shell
when=2020-06-09T14:22:51.405679+00:00 level=info pid=<0.527.0> at=supervisor:report_progress/2:1419
```

Reply:
If the formatter can be configured to include the module and function names automatically, then log lines can be further simplified to:

```suggestion
      ?LOG_INFO(#{
	 log => event, text => "Starting the UDP and TCP supervisor"
	}),
```